test/unittest/test_lib_utils.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+import socket
+
+from hypothesis import given, settings
+from hypothesis.strategies import integers
+
+import lib.utils as utils
+
+
+class TestUtils(unittest.TestCase):
+    def test_extract_schema_from_snapshot(self):
+        snapshot_path = '00000000000000000003.snap'
+        v = utils.extract_schema_from_snapshot(snapshot_path)
+        # assertListEqual() is available since 3.1
+        is_equal = v == [u'version', 2, 3, 1]
+        self.assertTrue(is_equal)
```

Comment:
Nit: Can we use `assertEqual()`?

Reply:
Ah, missed the comment. However, nope, we can use `assertEqual()`: it is more general than `assertListEqual()`, but works for lists (on Python 2 too).

Reply:
done

Reply:
works fine:
```
Traceback (most recent call last):
  File "test/unittest/test_lib_utils.py", line 15, in test_extract_schema_from_snapshot
    self.assertEqual(v, [u'version', 2, 3, 1])
AssertionError: (2, 3, 1) != ['version', 2, 3, 1]
```